packages/ggit/src/output/output.ts

```diff
@@ -64,38 +66,23 @@ export interface InputValidationError {
  */
 export interface AllErrors {
   AllErrors: {
-    Errors: Error;
+    Errors: SoilMetricsError;
   };
 }
 
 /**
  *
  * @example
  */
-export interface Errors {
-  AllErrors: InputErrors;
-}
-/**
- *
- * @example
- */
-interface InputErrors {
-  InputValidationErrors: Error;
-}
```

Comment:
I removed this intermediate interface in favor of defining the expectation of `InputValidationErrors` key within the `InputValidationError` interface - similar to how the `AllErrors` interface works